instagram/src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+
+import './App.css';
+import CommentSection from './components/CommentSection/commentSection';
+import dummyData from './dummy-data';
+import PostContainer from './components/PostContainer/postContainer';
+import SearchBar from './components/SearchBar/searchBar';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state ={ 
+      dummyData:[],
+      searchTerm: ''
+    };
+  }
+  componentDidMount() {
+    this.setState({ dummyData: dummyData });
+  }
+
+  searchFunction = event => {
+    event.preventDefault();
+    console.log('Nice Search');
+  }
+
+  render() {
+    console.log(this.state);
+    if (!this.state.dummyData.length) {
+      return <h4>Loading Posts...</h4>;
+    }    return (
+      <div className="App">
+        <div className='top-header'>
+          <SearchBar 
+            term={this.state.searchTerm} 
+            search = {this.searchFunction} 
+            handleSearchChange = {this.handleInputChange} 
+            content= {this.state.dummyData} />
+        </div> 
+        {this.state.dummyData.map((items, i) => 
+          <div className='posty' key = {i}>
```

Comment:
for your keys, use some  unique value of off item, like the thumbnail url or timestamp

using indexes creates bugs sometimes, so it's best avoided